src/documents/ModelTransformationAPI.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Model Transformation API
+layout: default
+category: Infusion
+---
+
+The *** model transformation *** framework is a core part of Infusion and allows you to transform change an input [model](FrameworkConcepts.md#model-objects) (JSON structure) based on a set of rules. 
+The result will be a new model, built according to the rules specified and the input model. Many kinds of transformation can also have their *inverses* computed
+and operated automatically.
+
+Any model transformation can be supplied as part of a [model relay](ModelRelay.md#explicit-model-relay-style) rule registered in a [fluid.modelComponent](ComponentConfigurationOptions.md#model-components)'s `modelRelay` 
+block - the system will then constantly operate the transformation rule whenever there is a model change to keep the linked model in sync. The user may also operate model transformation rules manually by use of the
+`fluid.model.transformWithRules` API.
+
+The best documentation for the currently available set of transformation functions is on the [GPII wiki](https://wiki.gpii.net/index.php/Architecture_-_Available_transformation_functions) but will
+in time be migrated here.
+
+## fluid.model.transformWithRules(source, rules[, options])
+
+* `model {Object}` The input model to transform - this will not be modified
```

Comment:
The signature above uses "source" as the first parameter, while the description calls it "model"